src/client.js

```diff
@@ -63,6 +63,7 @@ function Client(manager, name, config) {
 		config = {};
 	}
 	_.merge(this, {
+		awayMessage: "no users in the lounge",
```

Comment:
Now that this branch is leaving the status of patch, we should probably have a better default. Actually, empty by default since it will be disabled by default.